lib/idx/remediate.ts

```diff
@@ -211,11 +197,55 @@ export async function remediate(
         }
         return remediate(idxResponse, valuesWithoutExecutedAction, options); // recursive call
       }
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      else if (typeof action ==='string' && action in remediators!) {
```

Comment:
nit: `actions` is in type string[], no type check is needed here.

Reply:
Ha. Still getting used to TS